features/users/user_delete.feature

```diff
@@ -71,7 +72,8 @@ Scenario: Delete a user with a collection
     And 0 emails should be delivered
     And I should be logged out
     And a user account should not exist for "moderator"
-  When I go to the collections page
+  When all indexing jobs have been run
```

Comment:
This can be removed now that `I go to the collections page` runs the indexing jobs.